collective/dexteritytextindexer/converters.py

```diff
@@ -18,6 +18,7 @@
 from zope.schema.interfaces import ITuple
 
 import logging
+import six
```

Comment:
I believe `six` should also be declared as a direct dependency of this package in `setup.py`.

Reply:
Thanks for the review. I agree with your point.